ledger/participant-integration-api/src/main/resources/db/migration/postgres/V132__etq_prune_metadata_table.sql

```diff
@@ -0,0 +1,8 @@
+
+
+DELETE
```

Comment:
prune not needed entries. not sure we need this, if this is intra release, and we do not want to provide data continuity, we could probably skip it

Reply:
@pbatko-da @mziolekda WDYT? could we maybe remove this? Do we need data continuity inbetween the 2.6 track?
(I would be happy if we could, because Oracle has it's own flavor for delete with join, would be faster just to remove)

Reply:
discussed offline, we don't need data-continuity in between the official releases